### PR TITLE
feat:added signUp page

### PR DESCRIPTION
**Pull request by GuptaEesh**
Added sign up page with:
- email id registration
- password and repeat password fields
- terms and conditions acceptance.

ecomm7

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -1,7 +1,21 @@
...
+    let valPercent = (mySlider.value / mySlider.max) * 100;
+    mySlider.style.background = `linear-gradient(to right,var(--secondary-300) ${valPercent}%,var(--primary-200) ${valPercent}%)`
+}))
+
+//Terms and condition dialogue box.
+let termsInfo = document.querySelector('.terms-info');
+termsInfo.addEventListener('click', () => {
+    document.querySelector('.signup-page .login-dialogue .dialogue-container').style.display = "grid";
+})
+document.querySelector('.tertiary.border-between-button').addEventListener('click', () => {
```
Comment (FarhanMobashir) [inline index.js]: Keep the elements in a variable first and then add the event listener. That will keep the code clean

**Review by FarhanMobashir — COMMENTED** [review]
Added one comment. Everything else is fine

**Review by RakshithBhat03 — APPROVED** [review]
Everything looks good, please implement changes suggested by the other reviewer.

File: index.js (hunk 1 above)
Comment (GuptaEesh) [reply index.js]: Done. I unnecessarily used extra variables. Thanks for pointing it out, I have trimmed the fat.

**Comment by GuptaEesh** [comment]
Good suggestions were given by the reviewers. Implemented the suggestion, now merging the branch.

**Pull request merged**